test/utils.h

```diff
@@ -450,6 +453,10 @@ class NVFuserTest : public ::testing::Test {
                 << "'" << std::endl;
     }
   }
+
+#ifdef USE_DISTRIBUTED
+  static Communicator comm;
```

Comment:
I think we should not change `NVFuserTest`, instead, we should add a
```C++
class MultiDeviceTest: public NVFuserTest {
...
};
```
It is recommended for each topic to use its own test classes, for example in:
https://github.com/NVIDIA/Fuser/blob/178ed873baed1ad9b8e9bc2070551f0bb1f23f1f/test/test_expr_sort.cpp#L25

Reply:
done